### PR TITLE
Adding new package edgetest 

### PR DESCRIPTION
**Pull request by fdosani**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

Pinging @conda-forge/help-python

### CODE REVIEW

**Comment by fdosani** [comment]
@ak-gupta if you don't mind commenting you are confirming being a maintainer. For myself I confirm myself as a maintainer.

**Comment by ak-gupta** [comment]
I confirm being a maintainer

**Comment by fdosani** [comment]
@gforsyth Hope you are doing well! I'm wondering if I could get a hand here. It seems like `win64` is failing, but looking at the logs it isn't clear why the RC != 0. Would you have any insights on this. I've tried building on a Windows VM, but not getting the same issue here. It does look like it completes but the RC is off which is causing the fail.

**Comment by gforsyth** [comment]
Hey @fdosani !  I wish I could be more help but I can't figure out what's going wrong here.  Maybe someone from @conda-forge/help-python can take a look?

I do think you can remove the `LICENSE` file from this PR -- so long as the recipe includes the name of the license file and it's available in the upstream source repo that is sufficient.

**Comment by fdosani** [comment]
> you can remove the `LICENSE` file from this PR

Thanks Gil! Hope you are doing good. We miss you!

**Comment by fdosani** [comment]
@conda-forge/help-python would I be able to get some help here. I've been trying a variety of things to get the windows build working. I can locally build without issues on my local windows machine, and mac/linux works fine too. Not sure if I'm hitting some corner case with my setup or not?

Thank you in advance!

File: recipes/edgetest/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 83dfe3a2b5b0f1fc3386b3f042bca06095d70bb4ba0c2b8eb9e2f1d129dc5132
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (carterbox) [inline recipes/edgetest/meta.yaml:15]:
```suggestion
  script: "{{ PYTHON }} -m pip install -vv --no-deps --ignore-installed ."
```

**Review by carterbox — COMMENTED** [review]
Your package is marked as `noarch: python`, but it includes a binary named `edgetest.exe`. That's what [conda-build is complaining about](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=422464&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=06d62fd2-ab49-5add-9ee5-83b6882d7d2b&l=1661).

For clarity, it looks like you are using a [console_script entry point](https://packaging.python.org/specifications/entry-points/#use-for-scripts), but the implementation of this feature depends on the platform (is incompatible with noarch). The platform agnostic equivalent would require users to call `python -m edgetest`.

**Comment by fdosani** [comment]
@carterbox Thank you, makes sense!

**Comment by fdosani** [comment]
@conda-forge/help-python Ready for review. 
Special thanks to @carterbox for helping with the debugging.

**Comment by carterbox** [comment]
@fdosani, you may observe unexpected behavior on Windows? You package is still marked `noarch`, so once it is transferred to its own feedstock, it will be built *once* on linux. The console script placed in `$CONDA_PREFIX/bin` will be bash for all platforms.

**Comment by fdosani** [comment]
> @fdosani, you may observe unexpected behavior on Windows? You package is still marked `noarch`, so once it is transferred to its own feedstock, it will be built _once_ on linux. The console script placed in `$CONDA_PREFIX/bin` will be bash for all platforms.

Gotcha. I'll remove the arch part, thanks for following up.

**Comment by fdosani** [comment]
@carterbox I think we are good now. 🤞 Thank you again for your help.

**Comment by isuruf** [comment]
@carterbox, no using `entry_points` and `noarch: python` is the correct way. conda-build will fix it.

**Comment by fdosani** [comment]
> @carterbox, using `entry_points` and `noarch: python` is the correct way. conda-build will fix it.

@isuruf Would you recommend reverting back?

**Comment by isuruf** [comment]
> @isuruf Would you recommend reverting back?

yes

**Comment by fdosani** [comment]
@isuruf and @carterbox thank you again for the help. Think it is good now.

**Comment by isuruf** [comment]
@conda-forge-admin, lint

**Pull request merged**